full-service/src/json_rpc/txo.rs

```diff
@@ -11,17 +11,15 @@ use serde_derive::{Deserialize, Serialize};
 /// different statuses and types in relation to those accounts.
 #[derive(Deserialize, Serialize, Default, Debug, Clone)]
 pub struct Txo {
-    /// String representing the object's type. Objects of the same type share
-    /// the same value.
-    pub object: String,
-
     /// Unique identifier for the Txo. Constructed from the contents of the
     /// TxOut in the ledger representation.
     pub id: String,
 
-    /// Available pico MOB for this account at the current account_block_height.
-    /// If the account is syncing, this value may change.
-    pub value_pmob: String,
+    /// the txo's value
+    pub value: String,
+
+    /// the txo's token id
+    pub token_id: String,
```

Comment:
We may want to prefer the token name from enum as opposed to the int

Reply:
Are all tokens expected to have a name? I haven't looked into that yet, but if so I'm assuming it would be available via a network call or in the MC lib?

Reply:
Yeah, it's in the submodule, in the protos as `KnownTokenId` https://github.com/mobilecoinfoundation/mobilecoin/blob/master/api/proto/external.proto#L136

And defined in mc-transaction-core in the tokens module as implementing the `Token` trait:
https://github.com/mobilecoinfoundation/mobilecoin/blob/master/transaction/core/src/token.rs#L24

Reply:
The value of that is still an integer, though. Are you suggesting implementing a conversion within Full Service to map enum values to and from? If so, I would like to do that in a following update as that seems like purely an extension and won't change the API endpoints, will just substitute a stringified Int for a known token name.